setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import find_packages, setup
 
 
-version = '8.0.0'
+version = '8.0.1'
```

Comment:
Looks like everything went ok I've checked the release here https://pypi.python.org/pypi/django-user-management